services/queue/src/monitor.js

```diff
@@ -4,4 +4,46 @@ const manager = new MonitorManager({
   serviceName: 'queue',
 });
 
+manager.register({
+  name: 'resolvedQueuePoll',
+  type: 'resolved-queue-poll',
```

Comment:
This name's a little confusing, since the log format is used for the resolved queue but also the deadline queue and the claim queue.  Maybe `azureQueuePoll`?